css-brochure-form/css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 100px;
+  max-width: 500px;
+  max-height: 500px;
+}
+
+label {
+  margin: 10px;
+  text-align: left;
```

Comment:
Please make sure all of your properties do something, and remove them if they don't. Look out for "CSS Roulette"! Every line of styling should be written with a clear purpose.